### PR TITLE
config: update TD gRPC bootstrap test with v0.17.0 and v0.18.0

### PR DESCRIPTION
**Pull request by arvindbr8**

### CODE REVIEW

File: tests/bootstrap_generator_test.py
Comment (sergiitk) [inline tests/bootstrap_generator_test.py:1]: - [ ] [grpc/java/master/branch/xds_k8s_lb](https://source.cloud.google.com/results/invocations/18e36c31-4bbb-47f4-afdb-1e73a9aa900a)

**Review by sergiitk — APPROVED** [review]

File: tests/bootstrap_generator_test.py
Comment (arvindbr8) [reply tests/bootstrap_generator_test.py:1]: This seems to have failed. Could someone help me understand why tho?

Comment (sergiitk) [reply tests/bootstrap_generator_test.py:1]: Bootstrap generator test passed: https://btx.cloud.google.com/invocations/18e36c31-4bbb-47f4-afdb-1e73a9aa900a/targets;showStatuses=failed,passed/grpc%2Fjava%2Fmaster%2Fbranch%2Fxds_k8s_lb%2Fbootstrap_generator_test;config=default/tests.

The one failing is unrelated. It's been broken for a few weeks now. The team is working on it.

Comment (arvindbr8) [reply tests/bootstrap_generator_test.py:1]: Thanks for checking @sergiitk

**Pull request merged**